lib/hooks/useFileInput.ts

```
@@ -0,0 +1,58 @@
+      if (previewUrl) URL.revokeObjectURL(previewUrl);
+
+      setFile(selectedFile);
+
+      const objectUrl = URL.createObjectURL(selectedFile);
+      setPreviewUrl(objectUrl);
+
+      if (selectedFile.type.startsWith("video")) {
+        const video = document.createElement("video");
+        video.preload = "metadata";
+        video.onloadedmetadata = () => {
+          if (isFinite(video.duration) && video.duration > 0) {
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)